### DISCUSSION TITLE
Help to use edit the configuration of PHP-CS-Fixer using Globally (manual) installation

### DISCUSSION THREAD

**Discussion by Maofree** [PHP-CS-Fixer/PHP-CS-Fixer#6895] [Q&A] [answered]
Hi
I want to use the "Globally (manual)" suggested here
https://cs.symfony.com/doc/installation.html

I've tried and it works, but it did just a thing, remove the spaces and some tabs, instead I want to replace array() with [] and other fixes if necessary

how can use a configuration file to tell to it to do other things?

I use VSC and I can use this extension but I don't know  if it is compatible with the Globally installation

I need to fix 100 modules with many php files, so I don't want to install it to each module, I prefer a global use

is it possible to install it into /usr/local/bin/ of my debian mini server, then set the path to that file into VSC's extension, but I'd like to use a terminal command to each module's folder

can you help me?

thanks

**Comment by Maofree** [comment, marked as answer]
`php-cs-fixer fix ./ --rules='{"array_syntax": {"syntax": "short"}}'`

I found the solution

**Comment by Maofree** [comment]
Hi
can you tell me what is the rule that does this?

require_once implode(DIRECTORY_SEPARATOR, [
dirname(FILE),
'classes',
'ContactFormClass.php',
]);

from
require_once implode(DIRECTORY_SEPARATOR, [
dirname(FILE), 'classes', 'ContactFormClass.php',
]);

and the same for the arguments of an array?
$a = ['ddd', 'eee'];
to
$a = [
'ddd',
'eee'
];

from here I cannot find any parameter that does this
https://mlocati.github.io/php-cs-fixer-configurator/#version:3.16

thanks

**Reply by Wirone** [reply]
1. Please don't ask new questions in already answered and closed question
2. Please use block code formatting for better readability
3. Please use `-vvv` option when running Fixer to see what rules are applied to each file
